example/lib/main.dart

```diff
@@ -34,7 +34,6 @@ void callbackDispatcher() {
         stderr.writeln("The iOS background fetch was triggered");
         break;
     }
```

Comment:
revert changes of this file

Reply:
Gerevert en opnieuw default dart FTM formatting gedaan